volue_insight_timeseries/events.py

```diff
@@ -58,10 +59,8 @@ def fetch_events(self):
                             event.curve = self.curve_cache[event.id]
                         self.queue.put(event)
                         if sse_event.retry is not None:
-                            try:
+                            with contextlib.suppress(ValueError):
```

Comment:
Nice use of `contextlib`, but should we double check if there is some exception that could be handled here as well.

Reply:
Nice catch! I have today learnt that `int` can also throw `TypeError` in case the input is None. Adding this to the clause.